mriqc/interfaces/functional.py

```diff
@@ -141,7 +141,7 @@ def _run_interface(self, runtime):
 
         # FD
         fd_data = np.loadtxt(self.inputs.in_fd, skiprows=1)
-        num_fd = np.float((fd_data > self.inputs.fd_thres).sum())
+        num_fd = np.float64((fd_data > self.inputs.fd_thres).sum())
```

Comment:
This doesn't need to be cast at all. It will be a 64-bit int.

```suggestion
        num_fd = (fd_data > self.inputs.fd_thres).sum()
```

L147 will convert it to a Python int. L148 will convert it to a `float64` implicitly when dividing, and then cast the result explicitly to `float`.

Reply:
ok, thanks for the explanation @effigies.